src/constants.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Constants for the dashboard-aggregator service.
+ *
+ * @module constants
+ */
+
+export default {
+    DEFAULT_START_DATE_INTERVAL: "1 year",
```

Comment:
Since the query to list recently used apps is now limited to jobs that were launched by the authenticated user, limiting the number of matching jobs by start date is no longer as vital. I increased the default start date interval to 1 year.